package.json

```diff
@@ -162,6 +165,7 @@
 		"file-loader": "^0.11.0",
 		"find-remove": "^1.1.0",
 		"fs-extra": "^3.0.0",
+		"identity-obj-proxy": "^3.0.0",
```

Comment:
I'm unsure about this. It seems only to be for development and won't be used in the NPM module

Reply:
deleted